autoload/neomake/statusline.vim

```diff
@@ -168,15 +168,15 @@ function! s:formatter.format(f, args) abort
 endfunction
 
 
-function! s:running_jobs(bufnr) abort
+function! neomake#statusline#running_jobs(bufnr) abort
```

Comment:
@somini 
Do you have a need for this after all?
Seems like a good change by itself then, but I'd rather not expose things on the API if not necessary.

Reply:
With the new configuration this is not needed.